src/TestGen.ml

```diff
@@ -1,11 +1,49 @@
 open Core
 open Core.Quickcheck
 
-let for_type (t : Type.t) : Value.t Generator.t =
+let (--) lower upper =
```

Comment:
The design of test generator is that we pass parameters along with the type, so that we can specify list length distribution and int range.